src/commands/basemaps-github/make.cog.github.ts

```diff
@@ -161,6 +162,83 @@ export class MakeCogGithub {
     return tileSet;
   }
 
+  /**
+   * Get the collection.json url from vector target
+   *
+   * s3://linz-basemaps-staging/vector/3857/topographic/01HSF04SG9M1P3V667A4NZ1MN8/topographic.tar.co
+   * ->
+   * s3://linz-basemaps-staging/vector/3857/topographic/01HSF04SG9M1P3V667A4NZ1MN8/collection.json
+   */
+  getCollectionUrl(target: string): string {
+    const path = target.substring(0, target.lastIndexOf('/'));
+    return fsa.join(path, 'collection.json');
+  }
+
+  /**
+   * Prepare and create pull request for the aerial tileset config
+   */
+  async diffVectorUpdate(layer: ConfigLayer, existingTileSet?: ConfigTileSetVector): Promise<string> {
+    let md = '';
+    // Vector layer only support for 3857
+    if (layer[3857] == null) return md;
+    const newCollectionPath = this.getCollectionUrl(layer[3857]);
+    const newCollection = await fsa.readJson<StacCollection>(newCollectionPath);
+    if (newCollection == null) throw new Error(`Failed to get target collection json from ${newCollectionPath}.`);
+    const ldsLayers = newCollection.links.filter((f) => f.rel === 'lds:layer');
+
+    // Log all the new inserts for new tileset
+    if (existingTileSet == null) {
+      md += `New TileSet ts_${layer.name} with layer ${layer.name}.\n`;
+      for (const l of ldsLayers) {
+        md += `🟩 ${l['lds:name']} - version: ${l['lds:version']} features: ${l['lds:feature_count']}`;
+      }
+      return md;
+    }
+
+    // Compare the different of existing tileset
```

Comment:
could we add a comment saying there is generally only one layer here so this isnt going to fetch very much.